Cargo.lock

```diff
@@ -346,7 +346,7 @@ checksum = "bef41cbb417ea83b30525259e30ccef6af39b31c240bda578889494c5392d331"
 dependencies = [
  "libc",
  "ndarray",
- "num-complex 0.2.4",
+ "num-complex 0.4.5",
```

Comment:
I'm not sure why dependabot (well more likely cargo as I assume they're just calling `cargo update` with `--precise`) is having trouble with this. But using a newer num-complex here and for pyo3 is fine. I'll look at unifying this in parallel so we aren't building num-complex > 1 time.